gradle/libs.versions.toml

```diff
@@ -25,10 +29,12 @@ micronaut-reactor-http-client = { module = 'io.micronaut.reactor:micronaut-react
 micronaut-runtime = { module = 'io.micronaut:micronaut-runtime' }
 micronaut-tracing = { module = 'io.micronaut:micronaut-tracing' }
 micronaut-validation = { module = 'io.micronaut:micronaut-validation' }
+micronaut-docs-asciidoc-config-props = { module = 'io.micronaut.docs:micronaut-docs-asciidoc-config-props', version.ref = 'micronaut-docs' }
 
+javax-annotation-api = { module = 'javax.annotation:javax.annotation-api' }
 graal = { module = "org.graalvm.nativeimage:svm" }
```

Comment:
remove this and use `mn.graal` in dependency closures for build scripts. Ditto for logback (use `mn.logback`) and projectreactor (`mn.reactor`)